### DISCUSSION TITLE
Vite error when importing Shopify - "optimized info should be defined"

### DISCUSSION THREAD

**Discussion by nadavq** [Shopify/shopify-app-template-node#1063] [Q&A]
When I'm trying to import Shopify from shopify-api - i get this error:

Vite error, /node_modules/.vite/deps/@shopify_shopify-api.js?v=... optimized info should be defined.

Does anyone know how to solve this?

**Comment by devecomtoolshed** [comment]
By deleting my node_modules and installing "@shopify/app": "3.5.0" and "@shopify/cli": "3.5.0" I found I resolved that bug for other dependencies. Might be worth a try to fix. I also used "vite": "^3.2.0"
